testing/ncids-js-testing/public/inner-pages/multiple-autocomplete-fields.html

```diff
@@ -66,6 +66,7 @@
 	</head>
 	<body>
 		<a class="usa-skipnav" href="#main-content">Skip to main content</a>
+		<a name="top"></a>
```

Comment:
what is this?

Reply:
because git said it was an issue without it..